### PR TITLE
Update Flutter

### PR DESCRIPTION
**Pull request by damir-abdulin**

### CODE REVIEW

**Review by emmanuel-montoya — APPROVED** [review]

File: example/lib/main.dart [hunk 1]
```diff
@@ -1,10 +1,14 @@
+// ignore_for_file: prefer_int_literals
```
Comment (KirillSergeevich) [inline example/lib/main.dart:1]: why can't we fix it?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <4.0.0"
   flutter: ">=1.10.0"
 
 dependencies:
   flutter:
     sdk: flutter
+  very_good_analysis: ^5.0.0+1
```
Comment (KirillSergeevich) [inline pubspec.yaml:13]: it should be dev_dependency, right?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## 0.2.0
+Update pacakge to Flutter 3.10.
```
Comment (KirillSergeevich) [inline CHANGELOG.md:2]: typo

Comment (KirillSergeevich) [reply CHANGELOG.md:2]: also add these lines:
```
Dart 3 support
Style fixes
```

**Pull request merged**